apiary-metastore-listener/README.md

```diff
@@ -35,27 +35,81 @@ topic:
 The following shows an example JSON message representing a "CREATE_TABLE" event:
 
 	{
-      "protocolVersion":"1.0",
-      "eventType":"CREATE_TABLE",
-      "dbName":"some_db",
-      "tableName":"some_table"
+      "protocolVersion" : "1.0",
```

Comment:
Now that I'm actually looking at this and thinking about it properly I think @pradeepbhadani 's suggestion to have the colon immediately after the item (i.e. no space) and then a space after the colon does actually look the nicest. The examples here all appear to be formatted like that: https://json.org/example.html and https://en.wikipedia.org/wiki/JSON#Example (although one can obviously find examples formatted differently too)